zealot/src/zqd.ts

```diff
@@ -22,9 +23,18 @@ export type SearchWarnings = {
 
 export type SearchStats = {
   type: "SearchStats"
-  update_time: number
-  start_time: number
+  update_time: Ts
+  start_time: Ts
   bytes_read: number
+  bytes_matched: number
+  records_read: number
+  records_matched: number
+}
+
+export type SearchEnd = {
```

Comment:
I saw this in the stub you included, was the api sending these payload types before? or is this new?

Reply:
Yes it was sending these before, it's just the types that were incomplete.